sgx-isa/src/lib.rs

```diff
@@ -10,6 +10,9 @@
 //! convenient.
 //!
 //! [isdm]: https://www-ssl.intel.com/content/www/us/en/processors/architectures-software-developer-manuals.html
+#![feature(asm)]
+#![feature(lang_items)]
+#![feature(no_std)]
```

Comment:
These should all be dependent on the nightly feature being set. Also, we shouldn't need `lang_items` and `no_std`.